frontend/src/shared/components/Toast/store/createStore.ts

```diff
@@ -0,0 +1,24 @@
+type StoreSubscriber<T> = (state: T) => void;
+
+export interface Store<T> {
+  getState: () => T;
+  setState: (value: T | ((prev: T) => T)) => void;
```

Comment:
```suggestion
  setState: (value: T) => void;
```
`setState` 할 때 함수형 업데이트처럼 이전 값을 참조해야 하는 경우가 있나요??